content/tools/extensions/AzureDevOpsExtension.md

```diff
@@ -0,0 +1,63 @@
+<!--
+title: "Contrast Integration for Azure DevOps" 
+description: "Extension to integrate Contrast in your deployment workflow"
+tags: "tools azure devops integration extension deployment"
+-->
+
+
+Integrate Contrast in your deployment workflow with the Azure DevOps extension. The following instructions guide you through the steps to set up and configure the extension for your Contrast instance. 
+
+## Before You Start 
+
+Before you begin to set up the extension, make sure that you have the privileges to install a Microsoft extension. If you lack the necessary privileges, you can [request the extension](https://docs.microsoft.com/en-us/azure/devops/marketplace/request-extensions?view=azure-devops-2019) for a project.
+
+## Setup and Configuration
+
+### Step one
+
+* Follow [Microsoft's instructions](https://docs.microsoft.com/en-us/azure/devops/marketplace/install-extension?view=azure-devops-2019) to install the Contrast extension, **Contrast Integration**. 
+
+### Step two
+
+* Go to your **Project settings** at the bottom of you side bar.
```

Comment:
In what site/account does the user start on step two?

Reply:
Azure DevOps project page.